src/Index/reduxer.jsx

```diff
@@ -28,6 +29,20 @@ export const ReduxIndexBarChart = (() => {
   return connect(mapStateToProps, mapDispatchToProps)(IndexBarChart);
 })();
 
+export const ReduxEmailSignUpForm = (() => {
+  const mapStateToProps = () => {
+    if (components.index.emailSignUpForm) {
+      return { ...components.index.emailSignUpForm };
+    }
+    return {};
+  };
+
+  // ReduxEmailSignUpForm does not dispatch anything
+  const mapDispatchToProps = function mapDispatch() { return {}; };
```

Comment:
if nothing to dispatch, you don't have to have this

`connect()` can also work with only 1 arg